src/pages/Script/index.tsx

```diff
@@ -0,0 +1,254 @@
+
+  const { status, data: resp } = useQuery<AxiosResponse>(['scripts_general_info', codeHash, hashType], () =>
+    v2AxiosIns.get(`scripts/general_info`, {
+      params: {
+        code_hash: codeHash,
+        hash_type: hashType,
+      },
+    }),
+  )
+
+  const scriptInfo =
+    status === 'success' && resp
+      ? toCamelcase<Response.Response<ScriptInfo>>(resp?.data)!.data
+      : ({
+          id: 0,
+          scriptName: '',
+          scriptType: '',
+          typeId: '',
+          codeHash,
+          hashType,
+          capacityOfDeployedCells: '0',
+          capacityOfReferringCells: '0',
+          countOfTransactions: 0,
+          countOfDeployedCells: 0,
+          countOfReferringCells: 0,
+        } as ScriptInfo)
+  scriptInfo.scriptName = seekScriptName(scriptInfo.codeHash, scriptInfo.hashType)
+
+  useEffect(() => {
+    setCountOfTransactions(scriptInfo.countOfTransactions)
+    setCountOfDeployedCells(scriptInfo.countOfDeployedCells)
+    setCountOfReferringCells(scriptInfo.countOfReferringCells)
+  }, [scriptInfo.countOfDeployedCells, scriptInfo.countOfReferringCells, scriptInfo.countOfTransactions])
+
+  return (
+    <Content>
+      <div className={`${styles.scriptContentPanel} container`}>
+        <HashCardPanel isColumn={false}>
+          <ScriptsTitleOverview scriptInfo={scriptInfo} />
+        </HashCardPanel>
+        <Tabs
+          className={styles.scriptTabs}
+          activeKey={tab ?? 'transactions'}
+          onTabClick={key => {
+            const currentTab = tab ?? 'transactions'
+            if (currentTab === key) return
+
+            if (currentTab === 'deployed_cells') {
+              setPageOfDeployedCells(currentPage)
+            } else if (currentTab === 'referring_cells') {
+              setPageOfReferringCells(currentPage)
+            } else if (currentTab === 'transactions') {
+              setPageOfTransactions(currentPage)
+            }
+            if (key === 'deployed_cells') {
+              history.push(
+                `/script/${codeHash}/${hashType}/deployed_cells?page=${pageOfDeployedCells}&size=${pageSize}`,
+              )
+            } else if (key === 'referring_cells') {
+              history.push(
+                `/script/${codeHash}/${hashType}/referring_cells?page=${pageOfReferringCells}&size=${pageSize}`,
+              )
+            } else if (key === 'transactions') {
+              history.push(`/script/${codeHash}/${hashType}?page=${pageOfTransactions}&size=${pageSize}`)
+            }
+          }}
+          items={[
+            {
+              label: `${i18n.t('transaction.transactions')} (${localeNumberString(countOfTransactions!)})`,
+              key: 'transactions',
+              children: (
+                <ScriptTransactions
+                  page={currentPage}
+                  size={pageSize}
+                  updateCount={count => setCountOfTransactions(count)}
```

Comment:
The `updateCount` method is used to fetch `count` from the list component, and could be removed because `count` will be returned by the overview API

Reply:
> The `updateCount` method is used to fetch `count` from the list component, and could be removed because `count` will be returned by the overview API

In my opinion, after getting `count` from the overview API, the `count` will be updated every time the list API is returned. but it seems unnecessary when browse the page for a short time, I'll remove `updateCount`